pcsx2/ps2/Iop/IopHwWrite.cpp

```diff
@@ -248,6 +251,17 @@ static __fi void _HwWrite_16or32_Page1( u32 addr, T val )
 			//psxHu(addr) = val;
 		}
 	}
+	// ------------------------------------------------------------------------
+	// PS1 GPU access
+	//
+	else if( (masked_addr >= pgmsk(HW_PS1_GPU_START)) && (masked_addr < pgmsk(HW_PS1_GPU_END)) )
+	{
+		// todo: psx mode: this is new
+		DbgCon.Warning( "HwWrite32 to PS1 GPU? @ 0x%08X .. What manner of trickery is this?!", addr );
```

Comment:
In [Line 237](https://github.com/PCSX2/pcsx2/pull/1583/files#diff-418347f2c3fc26ef7d41a09e144108deR237) it is DevCon.Warning. Here it is DbgCon.Warning. Intended?